src/Compilers/VisualBasic/Portable/LanguageVersion.vb

```diff
@@ -156,7 +156,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     result = LanguageVersion.VisualBasic15_3
                 Case "15.5"
                     result = LanguageVersion.VisualBasic15_5
-                Case "16"
+                Case "16", "16.0"
```

Comment:
Let's add a corresponding test (adding a case to `LanguageVersion_TryParseDisplayString`).

Reply:
@jcouv I couldn't locate where the tests corresponding to this are. Sorry for that, can you please point me to where the corresponding test? Thanks.

Reply:
Look in `src\Compilers\VisualBasic\Test\CommandLine\CommandLineTests.vb`, which contains tests for LanguageVersion APIs

Reply:
Thanks! I added the test case for 16.0